Formula/libpsl.rb

```diff
@@ -0,0 +1,25 @@
+class Libpsl < Formula
+  desc "C library for the Publix Suffix List"
+  homepage "https://github.com/rockdaboot/libpsl"
+  url "https://github.com/rockdaboot/libpsl/releases/download/libpsl-0.14.0/libpsl-0.14.0.tar.gz"
+  sha256 "e8c794bcedf45c87dee4810570bc62ba7130217887e34e164e3eadc2259f2efb"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "md5sha1sum" => :build
+  depends_on :python2 => :build
```

Comment:
We have special dependencies `:python` and `:python3`.  What version of Python 2 is required?  If it's 2.7, you can do
```ruby
depends_on :python if MacOS.version <= :snow_leopard
```
If it's just whatever Python is on the system you shouldn't need it at all.